blogs/converter.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
```

Comment:
You can also use here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
so then for example you do
```
const { readdirSync } = require('fs');
```

Reply:
I'm not sure I understand this one, is it so you only pull reddirSync and not other functions from fs?

Reply:
don't I end up having to write this?

const {readdirSync} = require('fs');
const {readFileSync} = require('fs');
const {writeFileSync} = require('fs');